gradle/libs.versions.toml

```diff
@@ -3,35 +3,100 @@
 [plugins]
 
 com-android-application = { id = "com.android.application", version = "8.1.2" }
-
 org-jetbrains-kotlin-android = { id = "org.jetbrains.kotlin.android", version = "1.9.21" }
+com-android-library = { id = "com.android.library", version.ref = "agp" }
 
 [versions]
 
 kotlin = "1.9.21"
-##  ⬆ = "2.0.0-Beta1"
 
 androidx-databinding = "8.1.3"
+agp = "8.2.0"
+core-ktx = "1.9.0"
+##    ⬆ = "1.10.0-alpha01"
+##    ⬆ = "1.10.0-alpha02"
```

Comment:
Yo, is this the `refreshVersions` plugin doing some magic? This is an exciting prospect if it is